content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Hardware recommendations and considerations {#windows-hardware-recommendations}
+
+{{< note >}}
+The following hardware requirements are _recommended_ values and do not represent
+minimum requirements. Refer to [Production worker nodes Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/#production-worker-nodes)
```

Comment:
```suggestion
The following hardware requirements are _recommended_ starting values and do not represent
minimum requirements.  Depending on the requires for your workload these values may need to be adjusted. Refer to [Production worker nodes Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/#production-worker-nodes)
```

Or something like that, I am afraid of folks taking this at face value and coming to sig-windows saying we used your "recommended" values and things don't work.

Reply:
What about relaxing the wording a bit and instead of *recommendations* this documents is explicit that it presents *reasonable default* or *example* values, so this

> The following hardware requirements are _recommended_ values and do not represent
minimum requirements

could read

> The following hardware specifications should be considered as _reasonable default_ values and do not represent minimum requirements or production recommendations

Reply:
Going to combine both your feedback and draft something later today.

Reply:
I took both your feedback into account and reworded things. Please take a look @mloskot and @jsturtevant

Reply:
To me, as a user and reader, it looks good. Thx